packages/components/src/components/modal/modal.jsx

```diff
@@ -143,14 +152,20 @@ const ModalElement = ({
 
 ModalElement.defaultProps = {
     has_close_icon: true,
+    has_header_border: true,
+    is_item_align_centered: true,
```

Comment:
Are these default values affect existing modals? If yes then we should set the as `false`

Reply:
@matin-binary Modal header by default had a `header border` and all items in that were all centered, I wanted to remove the border and wanted to place items in left, so I set it `true` to make sure the existing modals remain same and we can set it to `false` for not having `header border` and the other.

Reply:
Great job, thanks for clarification 👍

Reply:
@aaron-binary  I didn't know that, so should I change the implementation in a way to turn the default values to false?

Reply:
Ideally we'd stick to whatever approach is common, but if unavoidable we can go ahead with this change.